src/OmniSharp.LanguageServerProtocol/Handlers/OmniSharpSignatureHelpHandler.cs

```diff
@@ -73,7 +73,7 @@ protected override SignatureHelpRegistrationOptions CreateRegistrationOptions(Si
             return new SignatureHelpRegistrationOptions()
             {
                 DocumentSelector = _documentSelector,
-                TriggerCharacters = new[] {".", "?", "["}
+                TriggerCharacters = new[] {"(", ",", "<"}
```

Comment:
Using my highly scientific method of [trigger character discovery](https://grep.app/search?q=override%20bool%20IsTriggerCharacter&filter[repo][0]=dotnet/roslyn&filter[path.pattern][0]=SignatureHelp), I determined that `{` is needed for initializer expressions and `[` is needed for element access expressions.

```suggestion
                TriggerCharacters = new[] { "(", ",", "<", "{", "[" }
```